qase-javascript-commons/src/reporters/testops-reporter.ts

```diff
@@ -370,9 +379,10 @@ export class TestOpsReporter extends AbstractReporter {
       return resultCreate;
     }
 
+    const rootSuite = this.rootSuite ? `${this.rootSuite}\t` : '';
```

Comment:
I guess, nested suites are not supported.

Reply:
The root suite is the first level suite. It can contain nested suites.